flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/HITSAlgorithm.java

```diff
@@ -273,4 +289,23 @@ public void sendMessages(Vertex<K, Tuple2<DoubleValue, DoubleValue>> vertex) {
 			return initVertexValue;
 		}
 	}
+
+	public static class AuthorityEdgeMapper<K, EV> implements MapFunction<Edge<K, EV>, String> {
```

Comment:
Before committing the newest codes, i have test the edge with `String` label and `boolean` label. And i found that `boolean` label is not fast than the `String` label. So finally i used the "hub" & "authority", also it's very intuitionistic.